packages/search-input/src/SearchResult/SearchResult.types.ts

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { AriaLabelProps } from '@leafygreen-ui/a11y';
+import { DarkModeProps } from '@leafygreen-ui/lib';
+import { InputOptionProps } from '@leafygreen-ui/internal-input-option';
+
+export type SearchResultProps = AriaLabelProps &
+  DarkModeProps &
+  InputOptionProps & {
+    /**
+     * The value of the result
+     */
+    children: React.ReactNode;
```

Comment:
we should probably have an explicit value prop. no?

Reply:
We had a discussion on the RFC , but didn't land anywhere. Feel free to weigh in! https://docs.google.com/document/d/1E_ACiuznG0_oubMDkDyDgwwq8qCJjojTBl96HgIpU9s/edit?disco=AAAAhCNvbbU

Reply:
nit: we could use `PropsWithChildren<...>` since we don't need special TSDocs for this prop

Reply:
`PropsWithChildren` sets `children` to optional. I feel like trying to get around that is more complex than declaring it here.
(That said, we could update `InputOptionProps` to use that utility type)